File: 110術科模擬試題/B2.py
#110商業技藝競賽模擬試題
#Problem B2 閏年
#Author: Yotrew Wing
#2021/10/18

n=int(input())
year=[]
for i in range(n):
    year.append(int(input()))
for i in range(n):
    if year[i] % 4 != 0:
        print("a normal year")
    elif year[i] % 400 ==0:
        print("a leap year")
    elif year[i] % 4 ==0 and year[i] % 100 !=0:
        print("a leap year")
    else:
        print("a normal year")

File: 110術科模擬試題/B2.java
/*
#https://github.com/yotrew/commercial_skill_competition
*/
import java.util.*;

public class B2 {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		String input;// = scanner.nextLine();
		input = scanner.nextLine();
		int n=Integer.parseInt(input);
		for(int i=0;i<n;i++){
			input = scanner.nextLine();
			int year=Integer.parseInt(input);
			if(year % 400 == 0)
				System.out.println("a leap year");
			else if( year % 4 ==0 && year % 100 !=0)
				System.out.println("a leap year");
			else
				System.out.println("a normal year");
		}
		
	}
}
